eventuals/http.h

```diff
@@ -958,7 +958,9 @@ struct _HTTP final {
 
               // Transform 'Request' headers to curl's linked list.
               for (const auto& [key, value] : request_.headers()) {
-                std::string header = key + ": " + value;
+                std::string header = key;
```

Comment:
I'm not in love with this switch to repeated calls to `append`: It seems nicer to instead switch to https://github.com/fmtlib/fmt

Reply:
I'm adding a comment for now, but I agree, we should probably use fmt library